### PR TITLE
Update/breath counting task

### PR DESCRIPTION
**Pull request by earcanal**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,51 +23,64 @@
...
+    }
 
-        jsPsych.init({
-             timeline: breath_counting_task_experiment,
-             display_element: "getDisplayElement",
-             fullscreen: true,
-             on_trial_finish: function(data){
-               addID('breath-counting-experiment')
-             },
+<<<<<<< HEAD
```
Comment (vsoch) [inline index.html]: hey looks like this shouldn't be here? This is from a merge.

**Comment by vsoch** [comment]
Are you concerned about keeping a working copy of the previous experiment? If so, this would be a good time to start a containershare equivalent table to ensure that the experiment here has all versions supplied as containers, and available on Github pages and docker hub, akin to this thing --> https://vsoch.github.io/containershare but with our experiment table instead.

**Comment by earcanal** [comment]
Good spot.  Thx.  I'm not concerned about keeping the previous version in this case.

**Comment by vsoch** [comment]
okay, we can do that for another experiment in the future. The other important thing is to have a change in an experiment trigger another testing of it in circle (the experiments repo) but I'm not totally sure how to do that yet, so I won't place the dependency on the PR here (unless you are interested in delving in?)

**Comment by earcanal** [comment]
I think that's above my pay grade :)

**Comment by vsoch** [comment]
haha, ok fair enough! I'm excited to do it, but definitely doesn't need to be now :) Happy to merge, and if there are issues with the experiment found via the table, they can still be addressed here.

**Pull request merged**